src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -47,6 +47,7 @@
 
 #include "boost/make_shared.hpp"
 #include "boost/format.hpp"
+#include <boost/algorithm/string.hpp>
```

Comment:
Please match the "" use from the surrounding code. If there's a case to be made for using <>, then make it and we can change everything, but let's keep consistency.